pii_recognition/pipelines/pii_validation_pipeline_test.py

```diff
@@ -89,34 +96,54 @@ def complex_scores():
         )
     )
 
+    # 3. test types occurred in multiple texts i.e. LOCATION
```

Comment:
are there still multiple locations here? It looks to me like there is only one in the scores

Reply:
Yes. Since we are calculating aggregated scores, case 4 and case 5 will be taken into the aggregation and they have LOCATION entities included, that is what I meant by multiple occurrences of LOCATION. The inline comment may not be clear, I will update it.